lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_methods.rb

```diff
@@ -7,18 +7,29 @@ class MiqAeServiceMethods
 
     SYNCHRONOUS = Rails.env.test?
 
-    def self.send_email(to, from, subject, body, content_type: nil, cc: nil, bcc: nil) # rubocop:disable Naming/MethodParameterName
+    def self.send_email(to, from, subject, body, *args, **kwargs)
```

Comment:
This method now accepts being called from `execute_with_user`, which will take the kwargs and append them to the args and send them along here.  Additionally, it accepts kwargs as it previously did by using any kwargs that are provided.

Reply:
Technically, this changes the interface of this method but doesn't break callers which used the original kwargs interface.